docs/streams/upgrade-guide.html

```diff
@@ -134,6 +134,24 @@ <h3 class="anchor-heading"><a id="streams_notable_changes" class="anchor-link"><
     </p>
 
     <h3><a id="streams_api_changes_370" href="#streams_api_changes_370">Streams API changes in 3.7.0</a></h3>
+    <p>
+        IQv2 supports <code>MultiVersionedKeyQuery</code> (introduced in <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-968%3A+Support+single-key_multi-timestamp+interactive+queries+%28IQv2%29+for+versioned+state+stores">KIP-968</a>)
+        that allows retrieving a set of records with the same specified key and different timestamps within a specified time range.
```

Comment:
`with the same specified` -> `for a given key` ?

`and a specified time range.`